async-http-client-backend/src/test/scala/sttp/client3/asynchttpclient/AsyncHttpClientHttpTest.scala

```diff
@@ -0,0 +1,7 @@
+package sttp.client3.asynchttpclient
+
+import sttp.client3.testing.HttpTest
+
+abstract class AsyncHttpClientHttpTest[F[_]] extends HttpTest[F] {
+  override protected def supportsNonAsciiHeaderValues = false
+}
```

Comment:
I decided to extract this class because it makes it clear that it's the `AsyncHttpClient` that does not support non-ascii header fields, which would not be apparent if this method was overridden in every `AsyncHttpClient*HttpTest`.

Reply:
That's a good idea, but probably there are more such common flags that are set in each test class individually which could be de-duplicated here

Reply:
I'll go through them and see what else I can extract.